docs/topics/plugin-resolve-issues.adoc

```diff
@@ -5,22 +5,53 @@
 [id='plugin-resolve-issues_{context}']
 = Resolving {ProductShortName} issues
```

Comment:
I think this should be either migration issues or just issues. MTA issues implies an issue with MTA, not an issue detected by MTA.